Source/santad/Logs/SNTEventLog.m

```diff
@@ -418,4 +420,23 @@ - (NSString *)originalPathForTranslocation:(santa_message_t)message {
   return [origURL path];  // this will be nil if there was an error
 }
 
++ (instancetype)logger {
+  static SNTEventLog *logger;
+  static dispatch_once_t onceToken;
+  SNTConfigurator *configurator = [SNTConfigurator configurator];
```

Comment:
Recommend removing the `configurator` variable. It's only used once inside the ``dispatch_once` and doesn't need to be grabbed on every call.

Reply:
Done